xep-0404.xml

```diff
@@ -337,85 +349,35 @@ This change means that the client will not be able to determine real JID of the
     <section2 topic='Sending Private Messages' anchor='usecase-user-private-messages'>
 
       <p>
-       Private Messages are used to provide communication with another channel participant through the MIX channel, where  the initiating  user does not the real JID of the channel participant.  A channel MAY support use of Private Messages.  Private messages are standard XMPP messages and MUST NOT be groupchat.   A message goes through a number of stages with different addressing.  This is set out in the following table.
+       Private Messages are used to provide communication with another channel participant through the MIX channel, where  the initiating  user does not know the real JID of the channel participant.  A message is addressed to the channel using the encoded JID of the client to which the message is being sent.   This is shown in the following example.
```

Comment:
It might be even if the user does know the real JID.
Saying it's addressed to the channel is misleading, it's addressed to the encoded JID of the user.
Messages are not generally sent to clients (although they could be) but to users.